marc_to_solr/lib/solr_deleter.rb

```diff
@@ -44,10 +46,10 @@ def delete_batch(batch)
       body = build_request_body(ids: batch)
 
       response = request_deletion(uri: uri, body: body)
-      return if response.status == 200
+      return unless response.nil? || response.status != 200
```

Comment:
maybe move this logic to a little helper method since it's used here and below. It would help readability to expand it to a full multi-line conditional.

Reply:
@hackartisan I think in your comment you meant to type: we are not ready to close issue #1389